src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -2,6 +2,7 @@
   (:require
     [quo2.core :as quo]
     [quo2.foundations.colors :as colors]
+    [quo.platform :as platform]
```

Comment:
`react-native.platform` should be used, quo shouldn't be used in status-im2

Reply:
resolved